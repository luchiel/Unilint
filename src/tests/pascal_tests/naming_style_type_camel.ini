[naming_styles]
class_naming_style = camel