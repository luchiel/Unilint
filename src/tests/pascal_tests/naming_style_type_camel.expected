23:3: name PascalClass is not of style camel
30:3: name underscore_type is not of style camel
31:3: name CAPS_UNDERSCORE_TYPE is not of style camel
